.gitignore

```diff
@@ -33,3 +33,6 @@ npm-debug.log
 
 # The built Escript
 /livebook
+
+# The output of elixir's lsp
+/.elixir_ls
```

Comment:
This should go to your [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/). :)